docs/src/snap/howto/cos-lite.md

```diff
@@ -0,0 +1,142 @@
+# Integrating with COS Lite
+
+**Canonical Kubernetes** includes the standard Kubernetes dashboard for
+monitoring your cluster. However, it is often advisable to have a monitoring
+solution which will run whether the cluster itself is running or not. It may
+also be useful to integrate monitoring into existing setups.
+
+To make monitoring your cluster a delightful experience, Canonical provides
+first-class integration between Canonical Kubernetes and COS Lite (Canonical
+Observability Stack). This guide will help you integrate a COS Lite
+deployment with a Canonical Kubernetes deployment.
+
+This document assumes you have a controller with an installation of
+Canonical Kubernetes. If this is not your case, refer to
+["Installing Canonical Kubernetes"][how-to-install].
+
+## Preparing a platform for COS Lite
+
+First, create a Microk8s model to act as a deployment cloud for COS Lite:
+
+```
+juju add-model --config logging-config='<root>=DEBUG' microk8s-ubuntu aws
+```
+
+We also set the logging level to DEBUG so that helpful debug information is
+shown when you use `juju debug-log` (see [juju debug-log][juju-debug-log]).
+
+Note: If you are unfamiliar with Juju models, the documentation can be found [here][juju-models].
+
+Use the Ubuntu charm to deploy an application named “microk8s”:
+
+```
+juju deploy ubuntu microk8s --series=focal --constraints="mem=8G cores=4 root-disk=30G"
+```
+
+Deploy Microk8s on Ubuntu by accessing the unit you created at the last step
+with `juju ssh microk8s/0` and following the [Install Microk8s][how-to-install]
+guide for configuration.
+
+> Note: Make sure to enable the hostpath-storage and MetalLB addons for Microk8s.
+
+Export the Microk8s kubeconfig file to your current directory after configuration:
+
+```
+juju ssh microk8s/0 -- microk8s config > microk8s-config.yaml
+```
+
+Register Microk8s as a Juju cloud using add-k8s (see ["juju
+add-k8s"][add-k8s] for details on the add-k8s
+command):
+
+```
+KUBECONFIG=microk8s-config.yaml juju add-k8s microk8s-aws
```

Comment:
```suggestion
KUBECONFIG=microk8s-config.yaml juju add-k8s microk8s-cloud
```
we used `microk8s` for the charm name and `microk8s-ubuntu` for the machine model.  i think it fits better to call this `microk8s-cloud` (otherwise you imply this is related to aws and it doesn't have to be).